aspnetcore/blazor/file-uploads.md

```diff
@@ -173,7 +188,7 @@ Because the example uses the app's [environment](xref:blazor/fundamentals/enviro
 
 The following example processes file bytes and doesn't send files to a destination outside of the app. For an example of a Razor component that sends a file to a server or service, see the following sections:
 
-* [Upload files to a server](#upload-files-to-a-server)
+* [Upload files to a server with client-side rendering (CSR)](#upload-files-to-a-server-with-client-side-rendering-csr)
```

Comment:
I always forget what our definitions for these terms are 😄 Is CSR limited to Blazor WebAssembly? Is that the term we prefer over "WebAssembly interactivity"? Including the term "rendering" here threw me off a bit because it's more about where the code is running than how the rendering happens. If this wording is consistent with what we already do elsewhere in the docs, then it's fine to leave it as-is.

Reply:
Totally fine if that's the established language!